sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -568,6 +568,22 @@
                 <title>Azure Core - Management</title>
                 <packages>com.azure.core.management*</packages>
               </group>
+              <group>
+                <title>Azure Core - Serializer - Avro - Apache</title>
+                <packages>com.azure.core.serializer.avro.apache*</packages>
+              </group>
+              <group>
+                <title>Azure Core - Serializer - Avro - Jackson</title>
+                <packages>com.azure.core.serializer.avro.jackson*</packages>
+              </group>
+              <group>
+                <title>Azure Core - Serializer - JSON - Gson</title>
+                <packages>com.azure.core.serializer.json.gson*</packages>
+              </group>
+              <group>
+                <title>Azure Core - Serializer - JSON - Jackson</title>
+                <packages>com.azure.core.serializer.json.jackson*</packages>
+              </group>
```

Comment:
These 2 groups can be removed.

Reply:
Whoops, copy and paste error

Reply:
Talked offline, decided it is sage to leave as it won't affect anything.